src/tests/components/SearchForm.test.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import SearchForm from "../../components/SearchForm";
+
+describe("SearchForm", () => {
+  it("renders correctly", () => {
+    const { asFragment, getByText } = render(<SearchForm />);
+
+    expect(asFragment()).toMatchSnapshot();
+    expect(getByText("Search")).toBeInTheDocument();
```

Comment:
Generally the snapshot test is its own thing and shouldnt have any other assertions in it. Not a major deal, just follows better with the engineering principle of separation of concerns